scripts/busybox-ping-test/container_ops.py

```diff
@@ -1,62 +1,78 @@
-#FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE 
-#AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
-#LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, 
-#OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+# MIT License
+# Copyright(c) 2020 Futurewei Cloud
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files(the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 from helper_functions import *
 from termcolor import colored
 
-def busybox_container_cleanup(ip, con):
+def busybox_container_cleanup(aca_node_ip, con):
     print("Cleaning up busybox container", con)
-    command = "ovs-docker del-ports br-int ".format(con)
-    output = run_command_on_host(ip, command)
-    command = "docker container stop ".format(con)
-    output = run_command_on_host(ip, command)
-    command = "docker container rm {} -f ".format(con)
-    output = run_command_on_host(ip, command)
+    command = "sudo ovs-docker del-ports br-int ".format(con)
+    output = run_command_on_host(aca_node_ip, command)
+    print("Cleanup task: ", command, "\n", output, "\n")
+    command = "sudo docker container stop {}".format(con)
+    output = run_command_on_host(aca_node_ip, command)
+    print("Cleanup task: ", command, "\n", output, "\n")
+    command = "sudo docker container rm {} -f ".format(con)
+    output = run_command_on_host(aca_node_ip, command)
+    print("Cleanup task: ", command, "\n", output, "\n")
+    command = "sudo ovs-vsctl del-br br-tun"
+    output = run_command_on_host(aca_node_ip, command)
+    print("Cleanup task: ", command, "\n", output, "\n")
+    command = "sudo ovs-vsctl del-br br-int"
+    output = run_command_on_host(aca_node_ip, command)
+    print("Cleanup task: ", command, "\n", output, "\n")
+    command = "sudo /usr/local/share/openvswitch/scripts/ovs-ctl start"
+    output = run_command_on_host(aca_node_ip, command)
+    print("Cleanup task: ", command, "\n", output, "\n")
```

Comment:
Is there a validation of output from run_command_on_host(), say, if it suceeded or failed and bail out if it had failed? This will prevent us going over more log liens than necessary.

Reply:
We don't check here. But in the next step we do check if ACA is up and running on the target nodes. If for any reason we don't find ACA running then test stops by giving a message that ACA is not running.